app/controllers/admin_raw_email_controller.rb

```diff
@@ -25,8 +25,9 @@ def show
             []
           else
             PublicBody.
-              where("lower(request_email) like lower('%'||?||'%')", domain).
-                order('name')
+              joins(:translations).
+                where("lower(request_email) like lower('%'||?||'%')", domain).
+                  order('name')
```

Comment:
Don't know what this is doing. Is this taking locale in to account?

If we need it, then:
- tests
- commit message explaining why we need the change.